scripts/linters/js_ts_linter.py

```diff
@@ -210,21 +211,32 @@ def _get_expressions_from_parsed_script(self):
 
         return parsed_expressions_in_files
 
-    def _compile_ts_file(self, filepath, dir_path):
-        """Compiles a typescript file and returns the path for compiled
-        js file.
+    def _get_compiled_ts_filepath(self, filepath):
+        """Returns the path for compiled ts file.
+
+        Args:
+            filepath: string. filepath of ts file
```

Comment:
Could you check the codebase to see if this is the right typeinfo? I thought it's usually either str/unicode (it should be what you get when you call `type(.)` on the argument).

Reply:
You're right - it's "str". Fixed